Source/Plugins/EditorBase/Properties/EditorBaseRes.Designer.cs

```diff
@@ -22,7 +22,7 @@ namespace Duality.Editor.Plugins.Base.Properties {
     [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
     [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
     [global::System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
-    public class EditorBaseRes {
+    internal class EditorBaseRes {
```

Comment:
Seems the generator made this internal.

After some searching there also seems to be a public variant of the generator. However if the resources are only used internally I don't see a reason to make them public and it would actually be cleaner to make them internal.

Reply:
The reasoning here was to allow user plugins to 1:1 use the builtin resources (especially icons) when desired - not sure this was ever used in that way.

I'm fine either way here, I think both ways have their points.